src/smartstream/derive/src/generator/map.rs

```diff
@@ -49,7 +49,7 @@ pub fn generate_map_smartstream(func: &SmartStreamFn) -> TokenStream {
                             let error = fluvio_smartstream::dataplane::smartstream::SmartStreamRuntimeError::new(
                                 &record,
                                 smartstream_input.base_offset,
-                                fluvio_smartstream::dataplane::smartstream::SmartStreamType::Filter,
+                                fluvio_smartstream::dataplane::smartstream::SmartStreamType::Map,
```

Comment:
there should be test to catch this issue.

Reply:
I opened an issue, can we handle later? https://github.com/infinyon/fluvio/issues/1356

Reply:
this kind of major issue can't be fell thru crack.  let's complete the test

Reply:
Added test